docs/reference/features.rst

```diff
@@ -0,0 +1,59 @@
+.. py:currentmodule:: PIL.features
+
+:py:mod:`features` Module
+==========================
+
+The :py:mod:`PIL.features` module can be used to detect which Pillow features are available on your system.
+
+.. autofunction:: PIL.features.pilinfo
+.. autofunction:: PIL.features.check
+.. autofunction:: PIL.features.get_supported
+
+Modules
+-------
+
+Support for the following modules can be checked:
+
+* ``pil``: The Pillow core module, required for all functionality.
+* ``tkinter``: Tkinter support.
+* ``freetype2``: FreeType font support via :py:func:`PIL.ImageFont.truetype`.
+* ``littlecms2``: LittleCMS2 support via :py:mod:`PIL.ImageCms`.
+* ``webp``: WebP image support.
+
+.. autofunction:: PIL.features.check_module
+.. autofunction:: PIL.features.get_supported_modules
+
+Codecs
+------
+
+These are only checked during Pillow compilation.
+If the required library was uninstalled from the system, the ``pil`` core module may fail to load instead.
+
+Support for the following codecs can be checked:
+
+* ``jpg``: (compile time) LibJpeg support, required for JPEG based image formats.
+* ``jpg_2000``: (compile time) OpenJpeg support, required for JPEG 2000 image formats.
+* ``zlib``: (compile time) ZLib support, required for ZLib compressed formats, such as PNG.
```

Comment:
```suggestion
* ``zlib``: (compile time) zlib support, required for zlib compressed formats, such as PNG.
```

Reply:
I went with "Zlib" for the first one, as it is the start of a sentence.